src/translations/en.json

```diff
@@ -107,6 +107,7 @@
     },
     "card": {
       "alarm_control_panel": {
+        "alarm_code": "Alarm Code",
         "code": "Code",
```

Comment:
Should we not use this existing key?

Reply:
I'm ok to do it. I've only tried to make the text same as before.

Reply:
I've already force pushed it: https://github.com/kukulich/home-assistant-frontend/commit/e668c3c63a0aecfe6269160d37958df2fa3eeea8

It looks there is problem in github with synchronization...